helix-core/src/indent.rs

```diff
@@ -360,21 +414,14 @@ fn query_indents(
                             Some(QueryPredicateArg::Capture(capt1)),
                             Some(QueryPredicateArg::Capture(capt2))
                         ) => {
-                            let get_line_num = |node: Node| {
-                                let mut node_line = node.start_position().row;
-                                // Adjust for the new line that will be inserted
-                                if let Some((line, byte)) = new_line_break {
-                                    if node_line==line && node.start_byte()>=byte {
-                                        node_line += 1;
-                                    }
-                                }
-                                node_line
-                            };
                             let n1 = m.nodes_for_capture_index(*capt1).next();
                             let n2 = m.nodes_for_capture_index(*capt2).next();
                             match (n1, n2) {
                                 (Some(n1), Some(n2)) => {
-                                    let same_line = get_line_num(n1)==get_line_num(n2);
+                                    let n1_line = get_node_start_line(n1);
+                                    let n2_line = get_node_start_line(n2);
+                                    log::debug!("n1 line: {}, n2 line: {}", n1_line, n2_line);
```

Comment:
Did you forget to remove this? Getting some debug logging for indent queries is not a bad idea in my opinion, but the text should be a bit more descriptive (maybe also just with `log::trace!` so it doesn't clutter the log too much).

Reply:
Oh yeah, I forgot to remove it. I'll take care of that.

Reply:
I think this still needs to ne adressed